packages/terminal-extension/src/index.ts

```diff
@@ -401,7 +402,7 @@ export function addCommands(
     label: 'Increase Terminal Font Size',
     execute: async () => {
       let { fontSize } = options;
-      if (fontSize < 72) {
+      if (fontSize && fontSize < 72) {
```

Comment:
It is unclear to me what should be the correct behavior when `fontSize` is undefined. This might be a fix or a break...

Reply:
No-op is fine I think